### PR TITLE
Check for loaded FB

### PR DESCRIPTION
**Pull request by iefserge**
R = @ellell 
T = patch

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,6 +4,13 @@ var _FB;
 var callbacks = [];
 
 module.exports = function (callback) {
+  if (window.FB) {
+    _FB = window.FB;
+    return process.nextTick(function () {
+      callback(null, _FB);
+    });
```
Comment (kesla) [inline index.js]: You don't need to do this here, this will be catched by the next if sentence.

Comment (iefserge) [inline index.js]: @kesla how? you mean `if (_FB) {` statement below?

Comment (kesla) [inline index.js]: yes.

we could even do `_FB = _FB || window.FB` as a one-liner :)

**Comment by iefserge** [comment]
@kesla updated

**Comment by kesla** [comment]
Awesome

**Comment by iefserge** [comment]
@kesla don't have publish permission

**Comment by kesla** [comment]
@iefserge neither have I :/

**Comment by kesla** [comment]
@iefserge I reached out to @ellell and asked to give us access

**Pull request merged**